frontend/common/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -7,35 +7,37 @@ export interface BreadcrumbsProps {
 }
 
 const Breadcrumbs: React.FunctionComponent<BreadcrumbsProps> = ({ links }) => (
-  <div data-h2-display="b(flex)">
+  <div data-h2-display="base(flex)">
     {links.map((link, index) => (
       <Fragment key={link.title}>
         {index > 0 && (
           <span
-            data-h2-padding="b(right-left, xs)"
-            data-h2-display="b(flex)"
-            data-h2-align-items="b(center)"
+            data-h2-padding="base(0, x.25)"
+            data-h2-display="base(flex)"
+            data-h2-align-items="base(center)"
           >
             <ChevronRightIcon style={{ width: "1.4rem" }} />
           </span>
         )}
         {link.href ? (
           <Link
-            data-h2-display="b(flex)"
-            data-h2-align-items="b(center)"
+            data-h2-display="base(flex)"
+            data-h2-align-items="base(center)"
             href={link.href}
             key={link.title}
           >
-            {link.icon || ""} {link.title}
+            {/* {link.icon || ""} {link.title} */}
```

Comment:
Any reason why this code has remained commented out and the link.icon has been removed?

Reply:
The icon was used inconsistently and was cluttering the UI visually. I commented it out to review with Jerbo, but it can be removed now.

Reply:
```suggestion
```